docs/usage/loading-custom-python-code.md

```diff
@@ -0,0 +1,98 @@
+
+(loading-custom-python-code)=
+# Loading custom Python code
+
+Pyodide provides a simple API {any}`pyodide.runPython` to run Python code.
+However, when your Python code grow bigger, putting hundreds of lines inside `runPython` is not scalable.
+
+For larger projects, the best way to run Python code with Pyodide is:
+
+1. create a Python package
+1. load your Python package into the Pyodide (Emscripten) virtual file system
+1. import the package with ``let mypkg = pyodide.pyimport("mypkgname")``
+1. call into your package with ``mypkg.some_api(some_args)``.
+
+## Using wheel packages
+
+The most recommended way of serving custom Python code is making it a wheel package.
```

Comment:
```suggestion
The best way of serving custom Python code is making it a package in the wheel (.whl) format.
```

I think technically wheel is package format, https://packaging.python.org/en/latest/glossary/#term-Wheel and "wheel package" sounds a bit unexpected to me.